site/app/templates/autograding/PeerResults.twig

```diff
@@ -330,9 +344,25 @@ function switch_feedback_modes() {
                 {% endif %}
 
             {% endfor %}
-            
+            </br>
+            <span style="float: right" id="save_status"></span>
+            <b>Peer Feedback:</b>
+            <div>
+                </br>
+                <a class="btn cat-buttons" data-color="#008000" style="border-color: #008000; color: #008000;" id="peer_feedback_no_response_{{grader}}" onclick="set_peer_feedback('{{grader}}', '{{user_id}}', '{{gradeable_id}}','no_response', '>No Response');">No Response</a>
+                <a class="btn cat-buttons" data-color="#008000" style="border-color: #008000; color: #008000;" id="peer_feedback_thanks_{{grader}}" onclick="set_peer_feedback('{{grader}}', '{{user_id}}', '{{gradeable_id}}', 'thanks', 'Thank You!');">Thank You</a>
+                <a class="btn cat-buttons" data-color="#008000" style="border-color: #008000; color: #008000;" id="peer_feedback_helpful_{{grader}}" onclick="set_peer_feedback('{{grader}}', '{{user_id}}', '{{gradeable_id}}','helpful', 'This feedback was helpful to me');">This feedback was helpful to me!</a>
+                <a class="btn cat-buttons" data-color="#008000" style="border-color: #008000; color: #008000;" id="peer_feedback_detailed_{{grader}}" onclick="set_peer_feedback('{{grader}}', '{{user_id}}', '{{gradeable_id}}', 'detailed', 'This feedback was detailed, specific, and/or technical');">This feedback was detailed, specific, and/or technical</a>
+                <a class="btn cat-buttons" data-color="#800000" style="border-color: #800000; color: #800000;" id="peer_feedback_inappropriate_{{grader}}" onclick="set_peer_feedback('{{grader}}', '{{user_id}}', '{{gradeable_id}}', 'inappropriate', 'This feedback was inaccurate and/or inappropriate');">This feedback was inaccurate and/or inappropriate</a>
```

Comment:
Same as above, can we get the styles into a css stylesheet?

Reply:
Added into server.cs